### DISCUSSION TITLE
Can one use XYChart to create a dual axis plot?

### DISCUSSION THREAD

**Discussion by gonzoramos** [airbnb/visx#1235] [General]
Hi @visx team!

First of all, I love this project and would like to use it for all my data vis needs :) - I have a question I hope you can help me with: I want to create a chart like this one

![image](https://user-images.githubusercontent.com/826412/116327135-ebde5f80-a77a-11eb-92a3-83ff6b606535.png)

Do you know how I can do that with XYChart? If not, should I go from scratch using lower level @visx components?

Many thanks in advance!
-G

**Comment by sgianelli** [comment]
I'm trying to do something similar with a line atop a stacked bar chart right now and it seems like the main problem is no being able to specify 2 separate yScales at the XYChart level though I'm not sure how you would reconcile that wrt negative values in the data set. I feel like the chart from scratch is the best option.

**Comment by williaster** [comment]
currently this is not supported in `@visx/xychart`, as it only supports one x- and one y-axis. In terms of data vis "best practice," I would generally discourage charts with two y-axes as they are hard to interpret and you can often achieve a similar effect by stacking two charts with a shared x-axis range.

others have tried overlaying two `XYChart`s on top of each other, but one significant challenge is tooltip triggering. see these related discussions 
- https://github.com/airbnb/visx/discussions/1165
- https://github.com/airbnb/visx/discussions/1161

another option I thought of would be to transform the data of one of the lines (say the red one) into the units of the other (the blue one), and then render a separate `@visx/axis` with a custom (red) scale. that's a decent amount of work, though, so doing a chart from scratch might be the easiest route.

**Comment by williaster** [comment]
going to convert this to a discussion

**Comment by JamesMDunn** [comment]
@williaster is there any updates on this? Possible support for two y-axis?

**Reply by ianchanning** [reply]
It looks like the answer to [Is layering XYcharts on top of each other possible? #1161](https://github.com/airbnb/visx/discussions/1161) is still the current best option
